pkg/graph/schema.resolvers.go

```diff
@@ -53,12 +53,20 @@ func (r *modelPlanResolver) Collaborators(ctx context.Context, obj *models.Model
 func (r *mutationResolver) CreateModelPlan(ctx context.Context, input model.ModelPlanInput) (*models.ModelPlan, error) {
 	logger := appcontext.ZLogger(ctx)
 	principal := appcontext.Principal(ctx).ID()
+	principalInfo, err := r.service.FetchUserInfo(ctx, principal)
+	if err != nil { //if can't get user info, use EUAID as commonName
```

Comment:
Note this logic is likely to go away as we move away from storing user name in the database. This code address if there is a failed call to cedar to get the users first name here. If first name is kept in the DB, this should be revisited